rst/cartridge_admin.rst

```diff
@@ -549,10 +555,44 @@ Enabling automatic failover
 
 In a master-replica cluster configuration with automatic failover enabled, if
 the user-specified master of any replica set fails, the cluster automatically
-chooses the next replica from the priority list and grants it the active master
-role (read/write). When the failed master comes back online, its role is
-restored and the active master, again, becomes a replica (read-only). This works
-for any roles.
+chooses one of replica from the priority list and grants it the active master
+role (read/write). To learn more about details of failover work, see **LINK**
+
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Failover disabled (default)
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The leader is the first instance according to topology configuration.
+No automatic decisions are taken. You can manually change leader in failover
+priority list or call ``box.cfg{read_only = false}`` on any instance.
+
+To disable it:
```

Comment:
```suggestion
To disable failover:
```

Introductory lines for step-by-step instruction should be self-contained because readers tend to skip text parts when scanning the page.